### PR TITLE
Simple Fixes for ISO 8601 Dates, Newline and Logfile Entries

### PR DESCRIPTION
**Pull request by Bobberty**
Simple fix for ISO 8601 Dates by modifying the date format.

Simple fix for removing the extra new lines in the log file.
Not too sure why two new lines are required.

### CODE REVIEW

**Review by Bobberty — COMMENTED** [review]
Blew the 4th patch....
Reject if necessary.  The fix is simple.  Just a PITA because I let my fingers go on remote control.

File: Project Source Code/WinFIM.NET Service/Service1.cs [hunk 1]
```diff
@@ -111,10 +111,11 @@ private void OnTimer(object sender, ElapsedEventArgs args)
         protected override void OnStop()
         {
             string serviceStopMessage = Properties.Settings.Default.service_stop_message +
-                                        $": (UTC) {DateTime.UtcNow:yyyy/dd/MM hh:mm:ss tt}\n\n";
+                                        $": (UTC) {DateTime.UtcNow:
```
Comment (Bobberty) [inline Project Source Code/WinFIM.NET Service/Service1.cs]: Blew this change.  Let my fingers go on remote control
Reject if necessary.  I will fix properly.
Thanks.

Comment (mpowrie) [reply Project Source Code/WinFIM.NET Service/Service1.cs]: Thanks @Bobberty for submitting the PR! 
I'll wait for the updated patch for line 114...

**Comment by Bobberty** [comment]
Fixed the fix

File: Project Source Code/WinFIM.NET Service/Service1.cs (hunk 1 above)
Comment (Bobberty) [reply Project Source Code/WinFIM.NET Service/Service1.cs]: Fixed....

**Comment by Bobberty** [comment]
Yet another change.  This one will allow better filtering for logging.  This would ensure the file and directory entries are in the same format without jumping a river for parsing.

**Comment by Bobberty** [comment]
More changes to allow easy log parsing.

**Comment by Bobberty** [comment]
Yet more log file entry changes.

**Review by mpowrie — APPROVED** [review]
Many thanks for the updates!

**Pull request merged**